source/web/css/guidelines.css

```diff
@@ -0,0 +1,22 @@
+/* layout for MEI Guidelines */
+p, div.p {}
+
+span.att {
+    font-weight: bolder;
```

Comment:
The former font-weight in mei-screen was 700, which would correspond to "bold" (with default normal being 400, bolder would only be 500). Change intentional?

Reply:
I found the attributes to be too thick, so this is a personal choice. For comparison: TEI highlights them only by adding a `@`.

Reply:
I have no strict opinion on the boldness of attributes in the guidelines, so fine with me. Just wanted to make sure it is not a typo. As stated above, for such changes not primarily related to the title and desc of the PR, it would help in the future to either avoid them at all (and put them in a separate PR) or at least indicate such changes in the description, to give others a hint about what to expect.

Reply:
moved rule back to `mei-screen.css`